### PR TITLE
Added doc and fixed build script

### PR DESCRIPTION
**Pull request by brewsteropsdev**

### CODE REVIEW

File: README-b2.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+#B2 Ubuntu Specifics
+This is simply a fork of the official Ubuntu linux repository used to create the images that land on Docker hub. We use this fork to ensure images running on our cluster are built from a known, signed source. The only modifications made support tagging images unique to our registry.
+
+The only branch we care about is `dist-amd64`, and it should be used as the base.
+
+## Updating
+1. Make sure you have added teh upstream repo.
```
Comment (QuangHuyNguyen) [inline README-b2.md]: Just a minor typo:
`teh` should be `the` instead

File: README-b2.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+```
+git remote add upstream git remote add upstream https://github.com/tianon/docker-brew-ubuntu-core.git
```
Comment (QuangHuyNguyen) [inline README-b2.md]: Is this a duplication of "git remote add upstream"?

File: README-b2.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+git checkout dist-amd64
+git merge upstream/dist-amd64
+```
+4. Push the changes to this repo.
+```
+git push
+```
+
+## Building
+A build script has been created to build (as of this writing) LTS verions of Ubuntu 14, 16, and 18, as well as the latest 18 verion. Provide the base URL of the repository as an argument while running.
```
Comment (QuangHuyNguyen) [inline README-b2.md]: what is the difference between LTS version 18 and the latest of 18 version?

Comment (QuangHuyNguyen) [reply README-b2.md]: I think I got it :)  So 18 is base version of 18.04.  And latest is the latest updates version of release 18.

**Review by QuangHuyNguyen — APPROVED** [review]
Look good, other than a few typos 👍

File: README-b2.md (hunk 3 above)
Comment (brewsteropsdev) [reply README-b2.md]: @QuangHuyNguyen yep exactly - 18 is 18.04, and latest 18 is 18.10 (at the moment). I'll update with specific versions in the doc.

**Pull request merged**